bitnami/pinniped/templates/concierge/deployment.yaml

```diff
@@ -71,6 +71,8 @@ spec:
         {{- if .Values.concierge.initContainers }}
           {{- include "common.tplvalues.render" (dict "value" .Values.concierge.initContainers "context" $) | nindent 8 }}
         {{- end }}
+      hostNetwork: {{ .Values.concierge.hostNetwork }}
+      dnsPolicy: {{ .Values.concierge.dnsPolicy }}
```

Comment:
Can we do this similar to the other parameters? If they are defined then add them, otherwise don't.

Reply:
Added if-statements 👍